bin/scripts/lib/fonts.json

```diff
@@ -1,5 +1,16 @@
 {
   "fonts": [
+    {
+      "unpatchedName": "0xProto",
+      "RFN": false,
+      "patchedName": "0xProto",
+      "folderName": "0xProto",
+      "imagePreviewFont": "0xProto",
+      "linkPreviewFont": "0xProto",
```

Comment:
I'm not sure but is this not:

```suggestion
      "linkPreviewFont": "oxproto",
```

Interestingly the zero is replaced by letter o?!

![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/7d1f72ba-2f27-4992-9f93-44ca7d522ff4)

Reply:
Kind of strange... yes...